CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
> adheres to [Semantic Versioning]

Do we actually want this? If not, I'll remove this statement.

Reply:
I'm a fan, though what that means w/r/t dependencies _here_ should probably be spelled out (esp. since this isn't a standalone app).

Reply:
I think we're semver across our GT-related projects, so let's stay consistent.

Reply:
Makes sense, I'll leave it in. 

> what that means w/r/t dependencies here should probably be spelled out

Sorry... I'm not totally sure what you mean. Upstream or downstream deps or both? Could you expand just a bit?

Reply:
I was about what happens if we bump upstream deps.

Since deps are shared between this lib and the downstream project, changing a version here potentially has pile-on effects downstream.

Reply:
Ah. Yes. That makes sense. I would expect that any time we bump upstream deps, but the API of this lib remains backwards compatible, then we wouldn't release a new major version -- verified via tests and usage. If you're referring specifically to upstream GT libs, I'm less confident about that but by default I'd expect the same restrictions.  @jpolchlo do we have a precedent in the other GT libs?

Reply:
/shrug

Probably want input from @pomadchin or @echeipesh